.rubocop.yml

```diff
@@ -0,0 +1 @@
+inherit_from: .rubocop_todo.yml
```

Comment:
Having the inheritance here instead of in the .rubocop_todo.yml will prevent linters in editors to function correctly, and it will be almost pointless to run rubocop before committing.

For example, writing any new method will allow an ABC complexity of 52 (which is a lot). 

It'll also make people not see existing errors, as they won't show up during the development process, and so they likely won't be fixed.